curator/patterns/reviews.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Title: Reviews
+ * Slug: curator/reviews
+ * Categories: featured, gallery
+ */
+?>
+
+<!-- wp:columns {"verticalAlignment":null,"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%"} -->
+<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"id":433,"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/sundae.png" alt="" class="wp-image-433"/></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"10%"} -->
+<div class="wp-block-column" style="flex-basis:10%"></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"center","width":"45%"} -->
+<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:45%"><!-- wp:paragraph {"fontSize":"x-large"} -->
+<p class="has-x-large-font-size"><?php echo esc_html__( 'The Fisherman', 'curator' ); ?></p>
```

Comment:
`h3` spec matching the design and it would be more appropriate to use.

Reply:
Good call, updated to h3s.